pkg/storages/memory/storage.go

```diff
@@ -21,8 +21,10 @@ type TimeStampedData struct {
 	Size      int
 }
 
+var NewObjectTime = time.Now
```

Comment:
I guess we want to store the time the object has been put into memory storage? Anyway, I don't quite get what has been changed here.

Reply:
This is for tests. Before, objects stored in this memory received the `time.Now()` timestamp. But now it's possible to override this `NewObjectTime` func to make objects' timestamps static and be able to test it.

Reply:
Then it would be better to pass it through the storage constructor `NewStorage()`. Because now this logic is not straightforward.

Reply:
I though it's a common practice to mock time this way, but I can pass it through the contructor, OK